CONTRIBUTING.rst

```diff
@@ -0,0 +1,85 @@
+============
+Contributing
+============
+
+Contributions are welcome, and they are greatly appreciated! Every
```

Comment:
"Each one helps"   or something different instead of "Every little bit helps"

Reply:
Okay, I'll change it later.